etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -65,8 +65,8 @@ spec:
         - name: POSTGRES_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: postgres # Must match secret setup by postgres subchart or postgres-secret.yaml
-              key: postgresql-password
+              name: {{ .Values.global.postgresqlExistingSecretName | default "postgres" }} 
```

Comment:
Shouldn't this be `.Values.global.postgresql.postgresqlExistingSecretName`?

Reply:
I think you also have an extra space in front of `key` here, throwing things off.